sdk/js/src/utils/consts.ts

```diff
@@ -216,8 +223,13 @@ const TESTNET = {
   near: {
     core: undefined,
     token_bridge: undefined,
-    nft_bridge: undefined,
+    nft_bridge: undefined,    
```

Comment:
```suggestion
    nft_bridge: undefined,
```

Trailing whitespace.